File: src/my_project/easy_problems/from251to300/find_all_k_distant_indices.py
from typing import List 

class Solution:
    def findKDistantIndices(self, nums: List[int], key: int, k: int) -> List[int]:
        a = []
        for i, n in enumerate(nums):
            if n == key:
                a.append(i)
        
        res = []
        for i in range(len(nums)):
            isk = False
            for j in a:
                if abs(i-j) <= k:
                    isk = True
                    break
                    
            if isk:
                res.append(i)
        
        return res
        
        

File: my_project_typescript/easy_problems_typescript/from151to200/findAllKDistantIndices.ts
function findKDistantIndices(nums: number[], key: number, k: number): number[] {
    let a: Array<number> = []
    for (let i=0; i < nums.length; i++){
        if (nums[i] === key){
            a.push(i)
        }
    }

    let res: Array<number> = []
    let isk: boolean = false
    for (let i=0; i<nums.length;i++){
        isk = false 

        for (let j of a){
            if (Math.abs(i-j) <= k){
                isk = true 
                break 
            }
        }
        if (isk){
            res.push(i)
        }
    }

    return res

};